run_imi.sh

```diff
@@ -50,6 +50,9 @@ if ! "$isAWS"; then
     printf "\nActivating conda environment: ${CondaEnv}\n"
     eval "$(conda shell.bash hook)"
     conda activate $CondaEnv
+
+    # Set GEOS-Chem environment
+    GEOSChemEnv="$(pwd -P)/envs/Harvard-Cannon/gcclassic.rocky+gnu12.minimal.env"
```

Comment:
Any ideas on how to do this well? It would feel kind of awkward for a Harvard user to have to change it to `/path/to/their/imi/clone/envs/Harvard-Cannon/gcclassic.rocky+gnu12.minimal.env` every time given that the environment already exists in their cloned version.

Reply:
On second thought, I think it is fine to expect Harvard users to have it somewhere in their home directory (and they could just copy from `envs/Harvard-Cannon` if not). I'll make the change.